microcosm_flask/swagger/parameters/list.py

```diff
@@ -18,7 +18,7 @@ def parse_items(self, field: Field) -> Mapping[str, Any]:
         Parse the child item type for list fields, if any.
 
         """
-        return self.build_parameter(field.container)
+        return self.build_parameter(field.inner)
```

Comment:
Tracing through the code, `container` was removed, and `inner` looks to be a nearly identical replacement.